src/MinimalDelegation.sol

```diff
@@ -66,17 +66,9 @@ contract MinimalDelegation is IERC7821, ERC1271, EIP712, ERC4337Account, Receive
         ExecutionData memory executeStruct = ExecutionData({calls: calls});
 
         (bytes32 keyHash, bytes calldata signature) = wrappedSignature.unwrap();
-        IHook hook = HookLib.get(keyHash, HookFlags.VERIFY_SIGNATURE);
-
-        bool isValid;
-        if (address(hook) != address(0)) {
-            isValid = hook.verifySignature(_hashTypedData(executeStruct.hash()), wrappedSignature);
-        } else {
-            // Use default signature verification.
-            isValid = _verifySignature(_hashTypedData(executeStruct.hash()), keyHash, signature);
+        if (!_verifySignature(_hashTypedData(executeStruct.hash()), keyHash, signature)) {
```

Comment:
nit: but if we use it like this I think we should have a similar isValid name for better readability

then this is more clear that its "if NOT valid"